server/services/conditionsService.ts

```diff
@@ -24,11 +24,15 @@ export class ConditionsServiceFactory {
 }
 
 export class ConditionsService {
-  constructor(private readonly version: ConditionVersion) {}
+  private readonly builder: LicenceWithConditionsBuilder
+
+  constructor(private readonly version: ConditionVersion) {
+    this.builder = new LicenceWithConditionsBuilder(version)
```

Comment:
move from a direct dependency to a local field which is parameterised with the same version of conditions to use as the main conditions service